docs/configurations.md

```diff
@@ -234,31 +217,39 @@ scalar.db.username=<USERNAME>
 scalar.db.password=<PASSWORD>
 ```
 
-### Example 2
+### Configuration example #2 - App, ScalarDB Cluster, and database
 
-```
-[App (ScalarDB Library with gRPC)] ---> [ScalarDB Server (ScalarDB Library with Consensus Commit)] ---> [Underlying storage/database]
+```mermaid
+flowchart LR
+    app["App -<br />ScalarDB library with gRPC"]
+    cluster["ScalarDB Cluster -<br />(ScalarDB library with<br />Consensus Commit)"]
+    db[(Underlying storage or database)]
+    app --> cluster --> db
 ```
 
-In this configuration, the app (ScalarDB Library with gRPC) connects to an underlying storage/database through ScalarDB Server.
-This configuration is recommended for production use because ScalarDB Server implements the [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface, which enables you to take transactionally consistent backups for ScalarDB by pausing ScalarDB Server.
+In this example configuration, the app (ScalarDB library with gRPC) connects to an underlying storage or database (in this case, Cassandra) through ScalarDB Cluster, which is a component that is available only in the ScalarDB Enterprise edition.
 
-In this case, an example of configurations for the app is as follows:
+{% capture notice--info %}
+**Note**
 
-```properties
-# Transaction manager implementation.
-scalar.db.transaction_manager=grpc
+This configuration is acceptable for production use because ScalarDB Cluster implements the [Scalar Admin](https://github.com/scalar-labs/scalar-admin) interface, which enables you to take transactionally consistent backups for ScalarDB by pausing ScalarDB Cluster.
 
-# ScalarDB Server host.
-scalar.db.contact_points=<SCALARDB_SERVER_HOST>
+{% endcapture %}
 
-# ScalarDB Server port.
-scalar.db.contact_port=<SCALARDB_SERVER_PORT>
-```
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
-And an example of configurations for ScalarDB Server is as follows:
+The following is an example of the configuration for connecting the app to the underlying database through ScalarDB Cluster:
 
 ```properties
```

Comment:
The properties are not correct. Please update it as follows:

```
# Transaction manager implementation.
scalar.db.transaction_manager=cluster

# Contact point of the cluster.
scalar.db.contact_points=indirect:<SCALARDB_CLUSTER_CONTACT_POINT>
```

Also, we should also mention the following here:
```
For client configurations for ScalarDB Cluster, see the ScalarDB Cluster [client configurations (redirects to the Enterprise docs site)](https://scalardb.scalar-labs.com/docs/latest/scalardb-cluster/developer-guide-for-scalardb-cluster-with-java-api/#client-configurations).
```

Reply:
Thank you for catching that. I've fixed the properties and mentioned the link to the ScalarDB Cluster client configurations page.

Reply:
I thought that might have been the case, but I assumed the configurations for this part would still be similar to the configurations for ScalarDB Server in `3.8` and earlier😔

I've removed the other, unnecessary configurations here.

Reply:
Ah I just realized that we don't need the following configurations even for ScalarDB Server :(
https://github.com/scalar-labs/scalardb/blob/3.8/docs/configurations.md?plain=1#L259-L267

Reply:
Got it. Thank you for letting me know! I'll remove those configurations from `3.8` and earlier versions as well.